infra/gp-external-secrets-configuration/templates/cicd-global-store.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ClusterSecretStore
+metadata:
+  name: cicd-global-store
+spec:
+  provider:
+    vault:
+      auth:
+        kubernetes:
+          mountPath: kubernetes
+          role: cicd-admin-reader
+          serviceAccountRef:
+            name: operate-workflow-sa
+      path: development/cicd/
```

Comment:
stimmt das? derselbe path wie oben?

Reply:
ja, es ist eine Berechtigungsunterscheidung: der cicd-global-store hat Zugriff auf *alle* subpaths (in Vault), weil er im cicd-tools und cicd-eventbus Namespace über Projekte hinweg Secrets anlegt und wir Berechtigungen nicht anhand des Namespaces des Service Accounts einschränken können, was aber okay sein sollte nachdem auf die Namespaces nur wir Zugriff haben. Erklärung wird im Merge-Fall im Confluence nachgezogen

Reply:
passt